cetmix_tower_yaml/tests/test_yaml_export_wizard.py

```
@@ -0,0 +1,375 @@
+# Copyright (C) 2024 Cetmix OÜ
```

Pylint (4.0.3)

[error] Unrecognized option found: readme-template-url, manifest-required-authors, manifest-required-keys, manifest-deprecated-keys, license-allowed, valid-odoo-versions

(E0015)